mix.exs

```diff
@@ -65,8 +65,8 @@ defmodule Sentry.Mixfile do
         ],
         authors: ["Mitchell Henke", "Jason Stiebs", "Andrea Leopardi"]
       ],
-      xref: [exclude: [:hackney, :hackney_pool, Plug.Conn, :telemetry]],
-      aliases: aliases()
+      xref: [exclude: [Finch, Plug.Conn, :telemetry]],
+      aliases: [aliases()]
```

Comment:
```suggestion
      aliases: aliases()
```
@savhappy The `mix test.integrations` task in CI is unavailable because this was changed to a list when the `aliases/0` function already returns a list. Reverting back should get you running